tests/test_data_cleaning.py

```diff
@@ -75,3 +80,30 @@ def test_clean_and_merge():
         ["reference", "period"]
     )
     assert_frame_equal(actual_output, expected_output)
+
+
+def test_create_imputation_class():
```

Comment:
This unit test fails on local machine, passed on Github actions so unsure about the cause of this. Failed on dypes int32 vs int64

Reply:
Interesting one, i will double check my dev env and pandas numpy versions

Reply:
for reference this was developed with pandas  1.1.5   and numpy   1.19.5   python in cdh